package.json

```diff
@@ -22,7 +22,9 @@
     "test:depcheck": "dependency-check ./package.json",
     "test:lint": "eslint --ext .js,.jsx .",
     "imagemin": "imagemin assets/images/* assets/images",
-    "postinstall": "npm-run-all build:*"
+    "postinstall": "npm-run-all build:*",
+    "snyk-protect": "snyk protect",
+    "prepare": "npm run snyk-protect"
```

Comment:
This applies temporary patches that snyk has provided. see the bottom of the `.snyk` file

Reply:
Where do these two commands get run?

Reply:
prepare is part of the npm install process: https://docs.npmjs.com/misc/scripts#description

Reply:
K